### PR TITLE
GH-2371 GH-2390 rsx namespace

### PR DESCRIPTION
**Pull request by hmottestad**
Introduce the rsx namespace for our rsx:targetShape definition.

### CODE REVIEW

**Comment by abrokenjester** [comment]
I've pushed an update so that the published url for the spec page will be `https://rdf4j.org/shacl-extensions/` instead of `https://rdf4j.org/shacl-extensions.html`. Ideally, the actual rsx namespace url should be identical.

**Comment by hmottestad** [comment]
> I've pushed an update so that the published url for the spec page will be `https://rdf4j.org/shacl-extensions/` instead of `https://rdf4j.org/shacl-extensions.html`. Ideally, the actual rsx namespace url should be identical. 

I was actually playing around with it and had found that we could just have a shacl-extensions file (without .html) and have it redirect to the html file.

**Comment by abrokenjester** [comment]
> I was actually playing around with it and had found that we could just have a shacl-extensions file (without .html) and have it redirect to the html file.

I'm not hung up on this particular approach, it's just something I noticed when I tried your PR out locally (wanted to see what it looked like), and it was an easy fix. Happy for you to do it differently of course.

File: Dockerfile
Comment (abrokenjester) [inline Dockerfile:1]: There is no need for a docker container in this project. Just use hugo.

**Review by abrokenjester — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (hmottestad) [reply Dockerfile:1]: It's just so I can test it locally.

Comment (abrokenjester) [reply Dockerfile:1]: I understand that, but like I said you don't need docker (or apache) for that . Just run `hugo server` in the site dir and you will have a running server at localhost:1313. It even hot replaces so you can edit and immediately see the changes.

Comment (hmottestad) [reply Dockerfile:1]: Oh. I didn't know that. Thanks.

Comment (abrokenjester) [reply Dockerfile:1]: No worries. We should probably put that in the readme somewhere.

Comment (abrokenjester) [reply Dockerfile:1]: Btw ive started migrating the doc stuff to the main git repo. Shall I migrate this branch as well? You'll have to check it out fresh if I do but that way we can have the migration done quickly. Alternative is to wait until this is ready to be merged

Comment (hmottestad) [reply Dockerfile:1]: If you just leave it hanging here I can migrate it over when you are done with migrating everything else. And I can keep working on it this weekend.

**Pull request closed without merging**
